integration-tests/gradle/src/test/resources/list-extensions-with-managed-dependency-without-scope/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
```

Comment:
I didn't delete these as I'd be needing them for the second test, it was giving errors so I didn't include it right now, ignore these for now :)